components/inputs/input-text.js

```diff
@@ -107,8 +107,7 @@ class InputText extends InputInlineHelpMixin(PropertyRequiredMixin(FocusMixin(La
 			 */
 			pattern: { type: String },
 			/**
-			 * Placeholder text
-			 * @type {string}
+			 * @ignore
```

Comment:
If we want to make this "internal" so that search/date can still use it, but remove it elsewhere, we could rename it to something like `__placeholderInternalOnlyDoNotUse`?

Reply:
That would make sense to me. I'll create a story so that we do that once the `input-text` `placeholder` cleanup is done.